server/index.js

```diff
@@ -7,17 +7,23 @@ const Restaurant = require('../db/index.js');
 // let jsonParser = bodyParser.json();
```

Comment:
Hi Matt. I think your code looks good. You can clean your code by deleting any code you don't use (line 7, 8 , 10). Thank you.

Reply:
Done. Thanks!